src/main/java/core/basesyntax/Machines/Excavator.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.Machines;
+
+public class Excavator extends Machine{
```

Comment:
There should not be a redundant empty line after the class declaration. Please remove the empty line to follow the proper code style.